### PR TITLE
Add Initial Release Date and numbers of two frameworks

### PR DESCRIPTION
**Pull request by tonyqus**
Preview
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,17 @@ The repo tries to compare Semantic Kernel and LangChain to show the difference a
 | Tools     | Plugins (semantic functions + native function) | Custom components that can be reused across different apps |
 | Memory    | Memory                                         | Store context and embeddings in memory or other storage    |
 
+## Initial Release Date
+LangChain: Oct, 2022
+
+Semantic Kernel: Apr, 2023
```
Comment (formulahendry) [inline README.md]: Should be `Mar`? refer to https://devblogs.microsoft.com/semantic-kernel/hello-world/

Comment (tonyqus) [reply README.md]: I checked [this page](https://github.com/microsoft/semantic-kernel/releases?page=8). The initial release is Apr 26, 2023.

Comment (formulahendry) [reply README.md]: Check from site of nuget package, I think the first released version should be this one: https://www.nuget.org/packages/Microsoft.SemanticKernel/0.8.11.1-preview on 3/1/2023

Comment (tonyqus) [reply README.md]: make sense

Comment (tonyqus) [reply README.md]: Updated

**Pull request merged**